java/src/main/native/src/ContiguousTableJni.cpp

```diff
@@ -24,6 +24,11 @@ namespace {
 jclass Contiguous_table_jclass;
 jmethodID From_packed_table_method;
 
+#define GROUP_BY_RESULT_CLASS "ai/rapids/cudf/GroupByResult"
+jclass Group_by_result_jclass;
+jfieldID Group_by_result_groups_field;
+jfieldID Group_by_result_uniq_key_columns_field;
+
```

Comment:
I rarely see that the C++ JNI code is depending on a Java class. Such dependency is dangerous if the Java class was modified inadvertently which may cause silent breaking. But I don't know a better solution for this case.

Reply:
I see no documentation for these functions and had difficulty in figuring out what they are doing. Please add doxygen explaining what they are.